content/about/security-release-process/security-release-process.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Security Release Process
+description: Release process for Istio security vulnerabilities.
+weight: 35
+page_icon: /img/security-release.svg
+---
+
+Istio follows the following process when releasing patches for critical
+security vulnerabilities.
+
+## Security vulnerability disclosures
+
+Please refer to the [disclosure process](/about/security-vulnerabilities/).
+
+## Istio security team
+
+Security vulnerabilities should be handled quickly and privately.
+The primary goal of this process is to reduce the total time users are
+vulnerable to publicly known exploits.
+
+The Istio security team is responsible for organizing the entire response
+including internal communication and external disclosure. The team will need help
+from relevant developers and release leads to successfully run this process.
+
+The Istio security team consists of volunteers subscribed to the
+private [`istio-security-vulnerabilities@google.com`](mailto:istio-security-vulnerabilities@google.com)
+list.
+
+### Istio security team membership
+
+Membership to the Istio security team is based on roles defined below.
+
+#### Roles
```

Comment:
All of these roles should be listed as bullet items, not as individual sections.

Reply:
Done.